androidsignature/signature.go

```diff
@@ -0,0 +1,40 @@
+package androidsignature
+
+import (
+	"regexp"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/command"
+)
+
+const (
+	validSignatureMessage = "jar verified"
+)
+
+// Read ...
+func Read(path string) (string, error) {
+	cmd := signatureCheckCommand(path)
+	output, err := cmd.RunAndReturnTrimmedCombinedOutput()
+	if err != nil {
+		return "", err
+	}
+
+	var signature string
+
+	if strings.Contains(output, validSignatureMessage) {
+		// The signature details appear in the output in the following format:
+		// - Signed by "C=Aa, ST=Bbbbb, L=Ccccc, O=Ddddd, OU=Eeeee, CN=Fffff"
+		regex := regexp.MustCompile(`- Signed by "(.*?)"`)
```

Comment:
Wouldn't it be enough to simply remove the suffix (`- Signed by`) from the relevant line and return the rest?

Reply:
The part we are interested in is also between `""` but I still think your suggestion is better because I think it is easier to process prefix and suffix removal than the understand why there is an array with two items and only the last one is needed.